### PR TITLE
Openblas win

### PR DESCRIPTION
**Pull request by eolivi**
try to add openblas for windows, with appveyor test.

### CODE REVIEW

File: OpenMEEG/cmake/UseOpenBLAS.cmake [hunk 1]
```diff
@@ -3,7 +3,24 @@
 #------------------------------------------------------------
 
 if (USE_OPENBLAS)
-    find_package(OpenBLAS ${REQUIRED} MODULE)
+    if(WIN32)
+        # download Openblas
+        if (NOT EXISTS ${CMAKE_SOURCE_DIR}/OpenBLAS.zip)
+			    if (FORCE_BUILD_32BITS)
```
Comment (agramfort) [inline OpenMEEG/cmake/UseOpenBLAS.cmake]: @eolivi can you fix the indentation?

**Comment by agramfort** [comment]
LGTM if appveyor is happy... let's keep our fingers crossed...

**Comment by eolivi** [comment]
non pas encore.. that was a try

File: OpenMEEG/cmake/UseWin32dlls.cmake [hunk 2]
```diff
@@ -1,45 +1,54 @@
-IF (WIN32 AND ENABLE_PACKAGING)
+#FIND_LIBRARY(MSVCP80
```
Comment (eolivi) [inline OpenMEEG/cmake/UseWin32dlls.cmake]: I will completely redo this file in the next PR (on MKL).

**Comment by agramfort** [comment]
thx @eolivi

**Pull request merged**